lib/Doctrine/DBAL/Migrations/Configuration/Configuration.php

```diff
@@ -782,18 +783,18 @@ public function createMigrationTable()
         }
 
         $this->connect();
-        if ($this->connection->getSchemaManager()->tablesExist([$this->migrationsTableName])) {
+        $schemaManager = $this->connection->getSchemaManager();
+        if ($schemaManager->tablesExist([$this->migrationsTableName])) {
             $this->migrationTableCreated = true;
 
             return false;
         }
 
-        $columns = [
-            $this->migrationsColumnName => new Column($this->migrationsColumnName, Type::getType('string'), ['length' => 255]),
-        ];
-        $table   = new Table($this->migrationsTableName, $columns);
+        $schema = new Schema([], [], $schemaManager->createSchemaConfig());
+        $table = $schema->createTable($this->migrationsTableName);
+        $table->addColumn($this->migrationsColumnName, 'string', ['length' => 255]);
```

Comment:
Can you keep the getType here ?

Reply:
When I change `string` into `Type::getType('string')` I've getting an exception back, because `\Doctrine\DBAL\Schema\Table::addColumn` method expects type as string. Internally it's doing the same thing: https://github.com/doctrine/dbal/blob/master/lib/Doctrine/DBAL/Schema/Table.php#L321

Reply:
ok